<?xml version="1.0"?>

<launch>

<param name="Rb" type="double" value="2.5"/>
<param name="d" type="double" value="10.0"/>
<param name="c" type="double" value="36.0"/>
<param name="ref_vel" type="double" value="1.0"/>
<param name="n_robots" type="int" value="1"/>

<!--Run the stage simulator-->
<node pkg = "stage_ros" name = "stageros" type = "stageros" output = "screen" args="-d $(find dubins_seg)/worlds/map_1_dubin.world" respawn="true">
</node>

<group ns = "robot_0">
    <node pkg = "dubins_seg" name = "controller" type = "ros_dubin_node.py" args="0 0 20.000 0.000 0.000 90.000" output="screen" respawn="true">
    </node>
</group>

<!-- Communication manager args="N_robots"-->
<!-- <node pkg = "dubins_seg" name = "communication" type = "ros_comm_node.py" args="0 0 1 1 2" output="screen" respawn="true">
</node> -->

</launch>
